subsys/dfu/boot/mcuboot.c

```diff
@@ -349,6 +353,7 @@ int boot_read_bank_header(u8_t area_id,
 	return 0;
 }
 
+#ifdef CONFIG_MCUMGR
```

Comment:
I don't understand. These are static and there are no functions by the same name anywhere else in the tree. Why is this necessary?

Reply:
Missed the thread on this.

Can we just rename them in this file to not conflict or something, since they're static? No need for extra ifdefing.

Reply:
yes, renaming is probably a better option, I thought there might be some purpose behind the same names. Btw, we have functions defined in the is file that are used no where in the tree. dead code basically, i.e `boot_read_bank_header`

Reply:
> since they're static?

btw, the main culprit is not static but if we ifdef it, the others will be become 'unused', hence why they all were ifdef'ed

Reply:
> yes, renaming is probably a better option. I thought there might be some purpose behind the same names.

That is my opinion as well, so will approve. Name were the same as function does the same as in mucboot (and had no better name before) - only that.